sdk/include/psdk/ntdddisk.h

```diff
@@ -408,9 +408,11 @@ typedef struct _PARTITION_INFORMATION_GPT {
 } PARTITION_INFORMATION_GPT, *PPARTITION_INFORMATION_GPT;
 
 typedef enum _PARTITION_STYLE {
-  PARTITION_STYLE_MBR,
+  PARTITION_STYLE_MBR = 0,
   PARTITION_STYLE_GPT,
-  PARTITION_STYLE_RAW
+  PARTITION_STYLE_RAW,
+  /* ReactOS custom partition handlers */
+  PARTITION_STYLE_BRFR = 128 /* Xbox-BRFR partitioning scheme */
```

Comment:
I would prefer to have this symbol defined separately in freeldr only... unless you think it may also be used in (e.g.) usetup or in ntoskrnl later (or in the HAL), in which case you should surround it within `#ifdef __REACTOS__ ... #endif` (as it is by the way done in some other parts of this file).

Reply:
@HeisSpiter any thoughts about that?

Reply:
Discussion from IRC:

```
<x86corez> I think it can and probably should be used in ntoskrnl later too
<x86corez> of course I investigated this as well, Xbox HAL have some hacks to override kernel functions for disk partitions
<@hbelusca> by the way x86corez, is it possible to use MBR-partitioned HDDs in XBOXes as long as the "operating system" is not the original MS XBOX OS ?
<x86corez> Yes it's just how software works, it isn't bound to the hardware
<@hbelusca> ok so this justifies the possibility of having the partitioning scheme used in xbox actually not inside xboxmach.c / xboxdisk.c in freeldr.
<x86corez> yep ^^)
```

Reply:
I think surrounding **just** the `PARTITION_STYLE_BRFR` within `__REACTOS__` would have been enough ;)

Reply:
That would leave the comma of previous enum entry before ifdef.

Reply:
and does it fail compiling in this case?

Reply:
I checked, it compiles just fine in this case, so everything is ok. :ok_hand: